test-plugins/org.yakindu.sct.generator.java.test/model/test.executionorder.sgen

```diff
@@ -0,0 +1,109 @@
+GeneratorModel for sctunit::java {
+	
+	
+	test ChildFirstExecutionHierarchy {
+		feature Naming {
+			basePackage = "org.yakindu.sct.generator.java.test"
+		}
+
+		feature Outlet {
+			targetProject = "org.yakindu.sct.generator.java.test"
+			targetFolder = "test-gen"
+		}
+		
+		feature StatechartNaming {
+			basePackage = "org.yakindu.scr"
+			libraryTargetFolder = "src-gen"
+		}
+	}
+
+
+	test ChildFirstLocalReactions {
+		feature Naming {
+			basePackage = "org.yakindu.sct.generator.java.test"
+		}
+
+		feature Outlet {
+			targetProject = "org.yakindu.sct.generator.java.test"
+			targetFolder = "test-gen"
+		}
+		
+		feature StatechartNaming {
+			basePackage = "org.yakindu.scr"
+			libraryTargetFolder = "src-gen"
+		}
+	}
+
+
+//	test ChildFirstOrthogonalReactions {
+//		feature Naming {
+//			basePackage = "org.yakindu.sct.generator.java.test"
+//		}
+//
+//		feature Outlet {
+//			targetProject = "org.yakindu.sct.generator.java.test"
+//			targetFolder = "test-gen"
+//		}
+//		
+//		feature StatechartNaming {
+//			basePackage = "org.yakindu.scr"
+//			libraryTargetFolder = "src-gen"
+//		}
+//	}
```

Comment:
The test model works but the test case not yet queue to missing sctunit features.
Was disabled in order to allow merging...